labels.md

```diff
@@ -0,0 +1,219 @@
+# Goals
+
+* synchronise labels between repos
+* make labels easier to set automatically
+* make labels clearer
+* make it easier to tell at a glance what is blocking PRs from landing
+* make colours more useful by making them more uniform
+
+# Naming scheme
+
+## Current issues
+
+* TiDB uses a `category/label` scheme, but with some exceptions
+  - closed, component, priority, status, type
+* TiKV uses a `C: label` scheme
+  - C (component), D (difficulty), P (priority), S (status), T (type)
+* PD uses a `category/label` scheme, but with some exceptions
+  - area, backport, kind, priority, 
+
+Bots work on all repos, but bot-readable labels are not distinguished.
+
+There is some duplication within a repo and some inconsistency about the category a label belongs to.
+
+## Proposal
+
+We use a `C: label` scheme for all repos (I don't feel strongly about the scheme, as long as there is just one. `C: label` is shorter, `category/label` is easier to understand). The categories are: component, priority, status, type, and difficulty.
+
+Where labels are similar across repos we pick one name and use that. For the status labels we use `waiting on CI`, `waiting on review`, `waiting on discussion`, and `waiting on author`, these are taken from the Rust repo where they have been found to be useful. We'll also keep some of the existing statuses where they are useful. The component labels are unique to each repo.
+
+# Colours
+
+Are generally inconsistent.
+
+## Proposal
+
+All labels in the same category have the same colour, taking the colour which is currently most popular.
+
+
+# Aternatives
+
+* Use `category/label`, `cat/label`, or `C-label` naming schemes
+* More categories (e.g., a bot category)
+* Allow labels without categories
+* Add emoji to highlight some labels or sub-categories
+* Separate labels for issues vs PRs
```

Comment:
Can github provide different labels for issues and PRs? It does looks ugly when we mixed labels used for issues and PRs.

Reply:
It can't, but we could namespace them ourselves, e.g., `I/type/Bug` or `PR/status/waiting on author`

Reply:
em, I don't like the format that adding a prefix 'I' and 'PR' to distinguish issues and PRs.